### PR TITLE
complete workout page (similar to health-content)

### PR DESCRIPTION
**Pull request by michael-greene-ey**

### CODE REVIEW

**Comment by ninohero** [comment]
@michael-greene-ey , you have merge conflicts.  you need to merge a/o rebase this branch with main branch

**Review by dmathiseney — APPROVED** [review]
Looks good 👍
Edit: I missed a file. Left a comment separately, but I don't think it's a blocker.

File: src/pages/Workouts.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.workout {
+  background-image: url("../assests/images/workout.jpg");
+  background-size: cover;
+  min-height: 62vh;
```
Comment (dmathiseney) [inline src/pages/Workouts.module.css]: What is 62vh doing? This is a little suspect. It's saying "make the height 62% of the screen size", which seems kind of strange. Does this work at different resolutions?

Comment (michael-greene-ey) [reply src/pages/Workouts.module.css]: i took it out. I don't need it anymore

**Pull request merged**